tensorflow/tools/graph_transforms/fold_old_batch_norms.cc

```diff
@@ -159,6 +159,9 @@ Status FuseScaleOffsetToConvWeights(const std::vector<float>& scale_values,
   NodeDef bias_add_node;
   bias_add_node.set_op("BiasAdd");
   bias_add_node.set_name(conv_output_name);
+  if (HasAttr(conv_node, "data_format")) {
```

Comment:
I don't see where is HasAttr defined. Maybe you can directly use something like:

```
if (!conv_node.attr().count("data_format")) {
  CopyNodeAttr(conv_node, "data_format", "data_format", &bias_add_node);
}
```

Reply:
Updated as requested